api/src/main/java/org/openmrs/module/commonreports/reports/PatientHistoryReportManager.java

```diff
@@ -101,9 +102,12 @@
 	@Autowired
 	BasePatientDataLibrary basePatientDataLibrary;
 	
+	@Autowired
+	private InitializerService inizService;
+	
 	@Override
 	public boolean isActivated() {
-		return super.isActivated();
+		return inizService.getBooleanFromKey("report.patientHistoryReportManager.active", false);
```

Comment:
The idea is that patient history should always be made available, unless otherwise stated. So I'm not sure I'm ok with this.